modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePaymentTerms.testcase

```diff
@@ -25,14 +25,51 @@ definition {
 		CommerceConfiguration.commerceInitializersTearDown();
 	}
 
-	@description = "[Please Update]"
+	@description = "Add a new payment term to Terms and Conditions."
 	@ignore = "Test Stub"
 	@priority = "4"
 	test AddANewPaymentTerm {
 		property portal.acceptance = "false";
 
 		// TODO COMMERCE-8279 AddANewPaymentTerm pending implementation
 
+		task ("Given a terms and coditions page.") {
+			CommerceNavigator.gotoPortlet(
+				category = "Order Management",
+				portlet = "Terms and Conditions");
+		}
+
+		task ("When a sales manager creates a new entry and is presented to the details page of the entry") {
```

Comment:
This should be two separate tasks like the test indicates. "When the sales manager creates a new entry" and "Then the Sales Manager is presented to the details page of the entry"